### PR TITLE
Revise filesystems page

### PR DESCRIPTION
**Pull request by willfurnass**
* Add info re different ways of handing chmod system calls on /shared
  filesystems
* Add more tables and cross-references
* Clarifications
* Formatting

### CODE REVIEW

**Comment by willfurnass** [comment]
@unixnation: any comments?

**Comment by willfurnass** [comment]
What would a 'backups' table look like for `/shared`?

File: hpc/filestore.rst [hunk 1]
```diff
@@ -33,68 +20,114 @@ The storage areas differ in terms of:
...
 
-**Backup policy:** ``/home`` has backup snapshots taken every 4 hours and 
-we keep the 10 most recent. 
-``/home`` also has daily snapshots taken each night, 
-and we keep 28 days worth, 
-mirrored onto a separate storage system.
++---------------------------+--------------------+---------------------------------------+
+| Frequency of snapshotting | Snapshots retained | Mirrored onto separate storage system |
++===========================+====================+=======================================+
+| Every 4 hours             | 10 most recent     | No                                    |
```
Comment (anthonybrookfield) [inline hpc/filestore.rst:47]: for /home snapshots - both the dailies and the 4 hourlies get sent to the backup storage system

File: hpc/filestore.rst [hunk 2]
```diff
@@ -33,68 +20,114 @@ The storage areas differ in terms of:
...
+| System   | Path                | Type   | Quota per user | Filesystem capacity | Shared between systems?                                | Network bandwith per link |
++==========+=====================+========+================+=====================+========================================================+===========================+
+| Bessemer | ``/fastdata``       | Lustre | None           | 460 TB              | No                                                     | 25Gb/s Ethernet           |
++----------+---------------------+--------+----------------+---------------------+--------------------------------------------------------+---------------------------+
+| ShARC    | ``/fastdata``       | Lustre | None           | 669 TB              | ShARC's fastdata filesystem is accessible from Iceberg | 100Gb/s (*Omni-Path*)     |
++----------+---------------------+--------+----------------+---------------------+--------------------------------------------------------+---------------------------+
+| Iceberg  | ``/fastdata-sharc`` | Lustre | None           | 669 TB              | ShARC's fastdata filesystem is accessible from Iceberg | 10Gb/s Ethernet           |
```
Comment (anthonybrookfield) [inline hpc/filestore.rst:118]: For most Iceberg worker nodes network bandwidth to lustre is only 1Gb/s

File: hpc/filestore.rst [hunk 3]
```diff
@@ -191,37 +235,49 @@ some applications can cause problems when accessing files/directories on ``/shar
...
+  a repository is simply moved to ``/shared/someplace`` from elsewhere on Bessemer/ShARC/Iceberg. 
   As a workaround you can tell git to not to track Linux permissions for a single repository using 
   ``git config core.filemode false`` or 
   for all repositories using ``git config --global core.filemode false``.
 
-The documentation for the ``/shared`` storage serivce includes information on:
+**Changing how attempts to change permissions are handled**: each ``/shared`` area can be configured so that
+
+1. Attempts to change file/directory mode bits fail (e.g. ``chmod +x /shared/mygroup1/myprogram.sh`` fails) (**default configuration per area**) **or**
+1. Attempts to change file/directory mode bits appear to succeed (e.g. ``chmod +x /shared/mygroup1/myprogram.sh`` does not fail but also does not actually change any permissions on the underlying file server) (**alternative configuration per area**)
```
Comment (anthonybrookfield) [inline hpc/filestore.rst:246]: this doesn't look to have formatted properly as separate items?

**Comment by willfurnass** [comment]
Made requested changes and:
 - Added table on backup schedule for /shared areas - info correct?
 - Revised section on checking quotas - suggest folks just use `df -h $STORAGE_AREA` as it works for arbitrary /shared areas too.
 - Revised section on accessing snapshots.

**Comment by willfurnass** [comment]
More changes following review comments:
 - fixed typos spotted by @desmfryan
 - defined snapshotting and mirrored backups in glossary
 - revised use of term 'backups' in filesystem page and added refs to glossary definitions
 - noted that *owners* of /shared areas need to request changes to how permissions handled

**Pull request merged**